.changeset/olive-lamps-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/base-radio-checkbox': patch
+'@twilio-paste/core': patch
+---
+
+[Base-radio-checkbox] Enable Component to respect element customizations set on the customization provider. Component now enables setting an element name on the underlying HTML element and checks the emotion theme object to determine whether it should merge in custom styles to the ones set by the component author.
```

Comment:
Super nit: Should this be hyphenated or should it read like a sentence? We've been capitalizing these so I think maybe this could be `BaseRadioCheckbox`?

Reply:
Good catch! Updated